docs/azure_jumpstart_arcbox/DevOps/_index.md

```diff
@@ -423,60 +423,46 @@ ArcBox deploys Kubernetes RBAC configuration on the bookstore application for li
 
-- To show Istio traffic split, open below windows.
-
-  - PowerShell running the below commands to show the bookbuyer pod logs.
-
-    ```powershell
-    $pod=kubectl --namespace bookbuyer get pods --selector=app=bookbuyer --output="jsonpath={.items..metadata.name}"
-    kubectl --namespace bookbuyer logs $pod bookbuyer -f | Select-String Identity:
-    ```
+- To show the Istio traffic split, open the below windows.
 
   - Click on the _Bookstore_ icon on the desktop to open bookstore applications.
 
-    ![Screenshot showing Bookstore desktop Icon](./capi_osm01.png)
-
-    ![Screenshot showing Bookstore Apps](./capi_osm02.png)
+    ![Screenshot showing Bookstore desktop Icon](./bookstore01.png)
 
-  - Move the browser tabs and PowerShell window, so the end result should look like this:
-
-    ![Screenshot showing Bookstore Apps and shell 01](./capi_osm03.png)
+    ![Screenshot showing Bookstore Apps](./bookstore02.png)
 
   - The count for the books sold from the bookstore-v2 browser window should remain at 0. This is because the current traffic split policy is configured as weighted 100 for bookstore as well because the bookbuyer client is sending traffic to the bookstore service and no application is sending requests to the bookstore-v2 service.
 
-    ![Screenshot showing Bookstore apps and shell 02](./capi_osm04.png)
-
-- In your fork of the “Azure Arc Jumpstart Apps” GitHub repository, open the _`traffic-split.yaml`_ file (_`/bookstore/osm-sample/traffic-split.yaml`_), update the bookstore weight to "75" and bookstore-v2 weight to "25" and commit the change.
+- In your fork of the “Azure Arc Jumpstart Apps” GitHub repository, open the _`istio-virtualservice.yaml`_ file (_`/bookstore/yaml/istio-virtualservice.yaml`_), update the weight to "75" and bookstore-v2 weight to "25" and commit the change.
 
-  ![Screenshot showing Bookstore repo Traffic split 01](./capi_osm05.png)
+  ![Screenshot showing Bookstore repo Traffic split 01](./bookstore05.png)
 
 - Wait for the changes to propagate and observe the counters increment for bookstore and bookstore-v2 as well.
 
-  We have updated the Service Mesh Interface (SMI) Traffic Split policy to direct 75 percent of the traffic sent to the root bookstore service and 25 percent to the bookstore-v2 service by modifying the weight fields for the bookstore-v2 backend. Also, observe the changes on the bookbuyer pod logs in the PowerShell window.
-
-  ![Screenshot showing Bookstore apps and shell GitOps and OSM 01](./capi_osm06.png)
+  We have updated the Istio virtual service to direct 75 percent of the traffic sent to the root bookstore service and 25 percent to the bookstore-v2 service by modifying the weight fields for the bookstore-v2 backend. Also, observe the changes on the bookbuyer pod logs in the PowerShell window.
```

Comment:
It's not clear where the PowerShell window is, so I would change it to the "kubectl command in the next step"